README.md

```diff
@@ -18,10 +18,10 @@ The framework is built for regression, integration & sanity testing. Includes te
 Refer to [Github documentation](https://help.github.com/articles/cloning-a-repository) on how to clone a repository. 
 
 ### Configure test environment
- 1. The test framework requires a distributed file system such as HDFS or MapR-FS to be configured. Hive and HBase are additional dependencies to run all tests (refer to the example in the Execute Tests section to exclude tests with dependencies).
- 2. The test framework expects Drill services to be setup on a clustered environment. Refer to [Drill documentation](http://drill.apache.org/docs/installing-drill-in-distributed-mode) for details on how to setup Drill.
- 3. Copy the `drillTestConfig` file from `framework/src/main/resources` to your home directory as `~/.drillTestConfig`. Edit suitably, following instructions in the file.
- 4. Source `.drillTestConfig`. Confirm `DRILL_HOME` and other required environment variables are set. 
+ 1. The test framework requires a distributed file system such as HDFS or MapR-FS to be configured. Some of the tests can also be run against a local file system. By default, it's configured to run against MapR-FS. You can change the default behavior by modifying conf/core-site.xml. Refer to conf/core-site.xml.example for settings.
+ 2. To run all tests, Hive and HBase needs to be installed and running. To exclude Hive and HBase tests, please refer to the example in the Execute Tests section.
+ 3. The test framework should be run on a Drill cluster node. Refer to [Drill documentation](http://drill.apache.org/docs/installing-drill-in-distributed-mode) for details on how to setup Drill.
```

Comment:
This is inaccurate. The original statement above is valid. The test framework can be configured to run on a client node or on one of the cluster nodes where Drill is configured.